metaci/plan/models.py

```diff
@@ -121,19 +121,27 @@ def check_push(self, push):
     
         return run_build, commit, commit_message
 
-SCHEDULE_CHOICES=(
-    ('daily', 'Daily'),
-    ('hourly', 'Hourly'),
-)
 
+class PlanRepositoryManager(models.Manager):
+    def get_queryset(self):
+        return super(PlanRepositoryManager, self).get_queryset().annotate(
+            alive=models.ExpressionWrapper(
+                models.Q(active=True) & models.Q(plan__active=True),
+                output_field=models.BooleanField()
+            )
+        )
```

Comment:
I think a comment explaining what's going on here would be helpful to our future selves.